src/components/NavHeader.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom';
+import '../styles/NavHeader.css';
+import React from 'react';
+// import { NavLink } from 'react-router-dom';
+// import planetImg from '../public/planet2.png';
```

Comment:
As requested by the previous code reviewer please consider removing the commented code from your code base to make your code more clean and professional. 🙏